internal/statestore/redis_indices_test.go

```diff
@@ -92,7 +95,8 @@ func TestExtractIndexedFields(t *testing.T) {
 			cfg := viper.New()
 			cfg.Set("ticketIndices", test.indices)
 
-			actual := extractIndexedFields(cfg, &pb.Ticket{Properties: test.ticket})
+			ticket := &pb.Ticket{Properties: test.properties, SearchFields: test.searchFields}
```

Comment:
Needed anymore? can we just not pass it in?

Reply:
Properties will be removed in a future PR, and this can collapse down again.  The actual:= line got too long with both properties and search fields.